var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -44,9 +44,12 @@ class Amrex(CMakePackage):
     variant('build_type', default='Release',
             description='The build type to build',
             values=('Debug', 'Release'))
+    variant('sundials', default=False,
+            description='Build AMReX with SUNDIALS support')
 
     # Build dependencies
     depends_on('mpi', when='+mpi')
+    depends_on('sundials@3.2.1: +ARKODE +CVODE', when='@develop +sundials')
```

Comment:
@balos1, Is sundials@4.0.1 not needed? This was the version I was attempting to test last week - based on what was discussed in the meeting.

Reply:
@balay AMReX itself builds fine with SUNDIALS 3.2.1. However, I am not sure if the applications that were discussed need 4.x or not.

Reply:
@balay Actually, after further investigating I think I need to explore the AMReX support of SUNDIALS a little more. It builds, but I am not sure if it is actually functional.